Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -3565,6 +3565,24 @@ internal void FailedToHashInputFile(LoggingContext loggingContext, string pipDes
             Message = "USN journal statistics for scheduler")]
         public abstract void JournalProcessingStatisticsForSchedulerTelemetry(LoggingContext context, string scanningJournalStatus, IDictionary<string, long> stats);
 
+        [GeneratedEvent(
+            (int)EventId.ProcessRetries,
+            EventGenerators = EventGenerators.TelemetryOnly,
+            EventLevel = Level.Verbose,
+            EventTask = (ushort)Tasks.Scheduler,
+            Keywords = (int)Keywords.UserMessage,
+            Message = "ProcessRetries PipsSucceedingAfterUserRetry:{0} and PipsFailingAfterUserRetry:{1}")]

```

Comment:
>{ [](start = 67, length = 1)

nit: add space between ':' and '{'. 

Also use named variable, i.e., instead of `{0}`, write `{pipsSucceedingAfterUserRetry}`. 

Similarly for `{1}`.